apps/docs/src/pages/components/stack.mdx

```diff
@@ -0,0 +1,99 @@
+import { HvStack, stackClasses, theme } from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("Stack", "core", stackClasses);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Playground
+  Component={HvStack}
+  componentName="HvStack"
+  controls={{
+    direction: {
+      defaultValue: "row",
+    },
+    spacing: {
+      defaultValue: "sm",
+    },
+    divider: {
+      defaultValue: false,
+    },
+  }}
+>
+
+    {Array.from({ length: 4 }).map((_, index) => (
+      <div
+        key={index}
+        style={{
+          display: "grid",
+          placeContent: "center",
+          width: 50,
+          height: 50,
+          backgroundColor: theme.colors.primary_20,
+        }}
+      >
+        {index + 1}
+      </div>
+    ))}
+
+</Playground>
+
+### Custom direction
+
+You can set the `direction` prop to different values depending on the breakpoint you're currently on. The example
+below shows a stack in a column format for breakpoints `xs` and `sm`, and in a row format for breakpoints `md` and above.
+
+```tsx live
+<HvStack direction={{ xs: "column", md: "row" }} spacing="xs" divider={false}>
+  {Array.from({ length: 4 }).map((_, index) => (
+    <div
+      key={index}
+      style={{
+        display: "grid",
+        placeContent: "center",
+        width: 50,
+        height: 50,
+        backgroundColor: theme.colors.primary_20,
+      }}
```

Comment:
the `backgroundColor` is will be rendered as `'var(--uikit-colors-primary_20)'`

![image](https://github.com/user-attachments/assets/8d53b23b-2d12-42be-9cc1-c5f02bd533d2)

we might as well just use Uno instead

Btw, why `primary_20`? It's used for actions (eg. hovers). Shouldn't we be using an `atmo*`?

Reply:
we could just use `HvPanel` instead